plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/CodeAssist.java

```diff
@@ -271,14 +280,10 @@ public void documentChanged(DocumentEvent event) {
                     p.apply(document);
                 }
 
-                ProposalApplyResult result = dtoFactory.createDto(ProposalApplyResult.class);
                 result.setChanges(changes);
                 Point selection = p.getSelection(document);
                 if (selection != null) {
-                    Region region = dtoFactory.createDto(Region.class);
-                    region.setOffset(selection.x);
-                    region.setLength(selection.y);
-                    result.setSelection(region);
+                    result.setSelection(DtoFactory.newDto(Region.class).withOffset(selection.x).withLength(selection.y));
```

Comment:
Take a look https://github.com/codenvy/codenvy-dev-resources/wiki/Formatting#align-when-multi-line -> Chained method calls. It looks better 
```
result.setSelection(DtoFactory.newDto(Region.class).withOffset(selection.x)
                               .withLength(selection.y));
```